_security/access-control/api.md

```diff
@@ -1336,6 +1374,55 @@ PUT _plugins/_security/api/nodesdn/<cluster-name>
 }
 ```
 
+### Update all distinguished names
+
+Bulk updates the given list of distinguished names.
+
+#### Path and HTTP methods
+
+```json
+PATCH _plugins/_security/api/nodesdn
+```
+{% include copy-curl.html %}
+
+#### Request fields
+
+| Field           | Data Type  | Description                                                                                                        | Required |
+|:----------------|:-----------|:-------------------------------------------------------------------------------------------------------------------|:---------|
+| op              | string     | The operation to perform on the action group. Possible values: `remove`,`add`, `replace`, `move`, `copy`, `test`.  | Yes      |
+| path            | string     | The path to the resource.                                                                                          | Yes      |
+| value           | array      | The new value to replace the property's value with.                                                                | Yes      |
+
+
+##### Example request
+
+```
+PATCH _plugins/_security/api/nodesdn
+[
+   {
+      "op":"remove",
+      "path":"/cluster1/nodes_dn/0"
+   }
```

Comment:
Is it possible to add a mock value to the example:

"value": "CN=Karen Berge,CN=admin,DC=corp,DC=Fabrikam,DC=COM", "CN=George Wall,CN=admin,DC=corp,DC=Fabrikam,DC=COM",

Reply:
yes, but since this is a remove operation we don't need it. If we want to add a sample value, I will change the example